### ISSUE TITLE
找不到samples.txt和predictions.txt

### ISSUE THREAD

**Issue by wshhja** [baidu/AnyQ#160]
你好，我在配置语义匹配模块的时候，按照simnet的文档进行训练和预测，训练的时候有一个worning：2019-05-13 12:49:52,200 - WARNING - line not match format in train file

在预测的时候，shell输出如下：`python paddle_simnet.py --task_type predict    --conf_file_path examples/cnn_pointwise.json
{u'loss': {u'class_name': u'SoftmaxCrossEntropyLoss', u'module_name': u'softmax_cross_entropy_loss'}, u'optimizer': {u'class_name': u'SGDOptimizer', u'learning_rate': 0.001}, u'dict_size': 3, u'num_threads': 4, u'test_file_path': u'data/test_pointwise_data', u'batch_size': 64, u'use_epoch': 0, u'train_file_path': u'data/train_pointwise_data', u'result_file_path': u'result_cnn_pointwise', u'net': {u'class_name': u'CNN', u'emb_dim': 128, u'filter_size': 3, u'hidden_dim': 128, u'num_filters': 256, u'module_name': u'cnn'}, u'epoch_num': 10, u'task_mode': u'pointwise', u'model_path': u'models/cnn_pointwise'}
2019-05-13 12:50:57,114 - INFO - start test process ...
2019-05-13 12:50:57,115 - WARNING - line not match format in test file`
没有其他的输出，在查看paddle_simnet.py后发现需要打开这两个文件samples.txt和predictions.txt，请问在哪里可以找得到呢？

**Comment by yinweichong** [comment]
samples.txt是需要预测打分的文件；
predictions.txt是保存打分结果的文件；
